pkg/helpers.go

```diff
@@ -25,6 +27,8 @@ type RegistryAddon struct {
 	} `yaml:"metadata"`
 }
 
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
```

Comment:
is `letters` only used in `randSeq()` function?

maybe it's a good idea to include it in the function scope like:
```go
func randSeq( ... ) ... {
  var letter = []rune( ... )
  runeSlice := make ...
  ...
}
```

in this way we avoid having global scope variable.